### ISSUE TITLE
Color-contrast error is thrown if there is a background-image on any parent element

### ISSUE THREAD

**Issue by nelson-w** [dequelabs/axe-core#3390]
**Product:** axe-core | axe Extension

**Expectation:** Detect if there is a color contrast issue

**Actual:** Throws error about color contrast, with impact of "unknown"

Code that causes the issue when run:
```
<body>
<div style="background-image: url();"><h1>Page not found</h1></div>
</body>
```

**axe-core version:** 4.3.5

**Browser and Assistive Technology versions:** Chrome Version 98.0.4758.82 (Official Build) (64-bit)

Does axe-core work with elements with background images? if it doesn't, is there a way to at least ignore this particular rule when it comes up against background-images(or comes up with an impact level of unknown) - not ignore the color-contrast rule altogether, as it's still useful for elements without background images, but just ignore it specifically for background-images.

**Comment by nelson-w** [comment]
@straker I'm running this in pa11y. Perhaps it's a pa11y issue where it's reporting this warning as an error. 

https://github.com/pa11y/pa11y/issues/623

**Comment by WilcoFiers** [comment]
Going to close this. This is an issue with pa11y, not axe-core.

**Comment by camtin** [comment]
In case people are still looking here years later, the issue is in the Pa11y Axe "runner" -- usually found in your /lib/runners/axe.js file.   Simply comment out call to the processIncomplete() function within the initial options.
